README.md

```diff
@@ -1,2 +1,19 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a) The algorithm looks for the same numbers from the list, but it must not be the same position.
+if it finds similar numbers, it returns true. Otherwise, returns that it didn't find anything.
```

Comment:
✔️ though the elements of the list could be any data type that can be compared for equality, not necessarily numbers